cms/templates/checklists.html

```diff
@@ -9,7 +9,7 @@
   from openedx.core.djangolib.js_utils import js_escaped_string, dump_js_escaped_json
 %>
 <%block name="title">${_("Checklists")}</%block>
-<%block name="bodyclass">is-signedin course uploads view-uploads</%block>
+<%block name="bodyclass">is-signedin course</%block>
```

Comment:
As a note, when we add in the link to the checklists page from the `Tools` drop down, we'd modify this line to have `tools` in it or something, right? I just wanted to make a mental note for myself.

Reply:
Yes, since that was part of the "Part 2" planned work I left it to be added during those tasks.